### ISSUE TITLE
Chamilo 1.11.8 and audio files

### ISSUE THREAD

**Issue by Elearner2018** [chamilo/chamilo-lms#2703]
### Current behavior / Resultado actual / Résultat actuel

I found a problem with audio files with auto play in the Learning Path. It looks like audio files are loaded twice causing stuttering effect. Chamilo first starts playing the audio file and then suddenly stopping it and then starting the audio file from the beginning again. With this second try the audio file is played fully.
Starting problems occur in courses created from backups and also in courses that are created in the platform. I tested this with the following browsers: Mozilla Firefox, Internet Explorer, Chrome and Opera. 
This did not happen in Chamilo 1.11.6

Audio files without auto play are played fully from the start to the end after the user has pressed the play button. This is ok.

### Expected behavior / Resultado esperado / Résultat attendu

Audio files with auto play are played immediately from the start to the end without reloading the file.

### Steps to reproduce / Pasos para reproducir / Étapes pour reproduire

Add audio file with auto play to a document in the Learning Path. 

### Chamilo Version / Versión de Chamilo / Version de Chamilo

Chamilo 1.11.8

**Comment by ywarnier** [comment]
Hi @Elearner2018 
Do you use the packaged version of 1.11.8 or the Git version (updated with "git pull"). I'm asking because there are some differences now, particularly in the audio files handling there.

**Comment by Elearner2018** [comment]
I used _Chamilo LMS 1.11.8 for PHP 7.0.x or later_ from the website https://chamilo.org/en/download/. Where to download the Git version?

**Comment by Elearner2018** [comment]
It seems that I should have downloaded from here: https://github.com/chamilo/chamilo-lms/releases/download/v1.11.8/chamilo-1.11.8-php7.tar.gz ?

**Comment by ywarnier** [comment]
Make sure you don't select autoplay when uploading the audio

**Comment by Elearner2018** [comment]
No, auto play was not selected before uploading the audio. It is always ticked after the download so I can't try this tip.

**Comment by ywarnier** [comment]
I *assume* you are using audio files in documents (rather than audio files attached to the item, which you can do in one of the option of the learning path). If you do so: could you edit the document in the documents tool ("learning paths" directory), right-click the audio, edit properties, then uncheck the "autoplay" and save?
(then try again)
